data_generators/xcspec_extractor.py

```diff
@@ -32,7 +32,7 @@ def xcspec_from_file(path):
 eval_bzl_path = sys.argv[4]
 xcode = os.path.dirname(os.path.dirname(developer_dir))
 xcode_plugins = os.path.join(
-    xcode, "Contents/PlugIns/XCBSpecifications.ideplugin/Contents/Resources"
+    xcode, "Contents/Developer/Library/Xcode/Plug-ins/XCBSpecifications.ideplugin/Contents/Resources"
```

Comment:
@thiagohmcruz I believe this was the source of my failure before. Guessing that Xcode 14.3 puts these in this new directory?

Reply:
I see, I don't know what others think but IMO this should be backwards compatible in case we need to run this for older versions? Can we conditionally change this value based on the version?

Reply:
This changed in 14 as well and we didnt add a flag, think its okay to bump this without it if the idea is the xccspecs are just generated by the latest supported Xcode